contracts/base/Escrow.sol

```diff
@@ -26,6 +36,12 @@ contract Escrow is TInitializable, EscrowInterface {
     using AddressLib for address;
     using AddressLib for address payable;
 
+    /**
+        @notice This is used to store values from within the dApp contracts.
+        @dev dApp contracts cannot have their own storage variables as they will overwrite the each other's  storage slots in the proxy.
```

Comment:
mmmm I think we had talked to add params in the dapp functions.

Reply:
ahh yes my bad.